src/common/mailingListServices.jsx

```diff
@@ -0,0 +1,145 @@
+import React from "react"
+import { useState } from "react"
```

Comment:
quotes, also, u should be able to combine these 2 into 1, can't u? `import React, { useState } from 'react'`